Classes/Form/Element/JsonFieldElement.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Mediatis\Formrelay\Form\Element;
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Utility\StringUtility;
+
+class JsonFieldElement extends \TYPO3\CMS\Backend\Form\Element\AbstractFormElement
```

Comment:
@mkarulin Is there a specific reason for extending AbstractFormElement instead of TextElement?

Maybe this would have some weird side-effects that you already experienced, but my approach would be to just update $this->data['parameterArray']['itemFormElValue'] and then call the parent render function. Is this not possible in this case?

Reply:
Good point. Didn't think of this at that moment.